.github/actions/generate-coverage/tests/conftest.py

```
@@ -0,0 +1,17 @@
+import pytest
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)